shapes/binning.yaml

```diff
@@ -23,3 +23,112 @@ analysis:
     ztt:    [0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
     noniso: [0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
     misc:   [0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+gof: 
```

Comment:
key und expression sind identisch... wofür ist die expression?

Reply:
expressions are removed now

Reply:
update: expressions are back in